docs/configure/integration.md

```diff
@@ -0,0 +1,315 @@
+      shouldExtractLiteralValuesFromEnum: true,
+      propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
+    },
+  },
+};
+```
+
+<div style="background-color:#F8FAFC">
+TODO: vet SEE DOCS link
+</div>
+
+|Field	                         |Framework	    |Description	                                                                          |Type      |
+|:-------------------------------|:------------:|:---------------------------------------------------------------------------------------:|:--------:|
+|**check**                       |All	        |optionally run fork-ts-checker-webpack-plugin	                                          |boolean   |
+|**checkOptions**	             |All	        |Options to pass to fork-ts-checker-webpack-plugin if it's enabled	                      |See docs  |
+|**reactDocgen**	             |React	        |which variant docgen processor to run	`'react-docgen-typescript'                        |N/A       |
+|**reactDocgenTypescriptOptions**|React	        |Options to pass to react-docgen-typescript-plugin if react-docgen-typescript is enabled. |See docs  |
+
+
+
+### Styling and CSS
+
+There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually it is best to try and replicate what your application does with styling in Storybook’s configuration.
+
+#### CSS-in-JS
+
+CSS-in-JS libraries are designed to use basic JavaScript. They often work in Storybook without any extra configuration. Some libraries expect components to be rendered in a specific rendering “context” (such as to provide themes) and you may need to add a [global decorator](../writing-stories/decorators#global-decorators) to supply it.
+
+#### Importing CSS files
+
+<div style="">
+TODO: check note on sb-storybook-6_0_more_feedback.txt for this
+</div>
+
+If your component files import their own CSS, Storybook’s webpack config will work unmodified with some exceptions:
+
+- If you are using a CSS precompiler, you may need to add a preset (such as the [SCSS preset](https://github.com/storybookjs/presets/tree/master/packages/preset-scss), or add a loader to Storybook’s Webpack config).
+- In Angular, you'll need to take special care how you handle CSS:
+    - Either [customize your webpack config](#extending-storybooks-webpack-config)
+    - Or use syntax to use a inline loader:
+        ```js
+            import '!style-loader!css-loader!./styles.css';
+        ```
+    
+
+To use your CSS in all stories, you simply import it in [`.storybook/preview.js`](./overview#configure-story-rendering)
+
+#### Adding webfonts
+
+If you need webfonts to be available, you may need to add some code to the [`.storybook/preview-head.html`](./story-rendering#adding-to-head) file. We recommend including any assets with your Storybook if possible, in which case you likely want to configure the [static file location](../api/cli-options).
```

Comment:
We talk about static files just below in this section, I think we link there here

Reply:
It's now properly linked.